### ISSUE TITLE
God's Prophetic Word (GPW) translation

### ISSUE THREAD

**Issue by erykjj** [erykjj/jwpublib#42]
Please let me know here if you can offer your help with translation on this project.

**Comment by LukaKru** [comment]
I can try to polish :)

**Comment by erykjj** [comment]
Hi @shonind, @JohnDoeDK, @ferrarantonio

Would you be insterested in trying this? It is only some 150 strings :-)

**Comment by shonind** [comment]
That looks fun and colourful 😄

**Comment by JohnDoeDK** [comment]
> Hi @shonind, @JohnDoeDK, @ferrarantonio
> 
> Would you be insterested in trying this? It is only some 150 strings :-)

Hi Eryk!

Looks like an interesting project, I'm in!

**Comment by ferrarantonio** [comment]
I am available to translate the text into Italian

**Comment by Goykew** [comment]
Witam serdecznie. ❤️Ja mogę tłumaczyć na 
Polski

**Comment by erykjj** [comment]
@shonind I sent you an invite to another repo. Have you seen it?

**Comment by erykjj** [comment]
> Witam serdecznie. ❤️Ja mogę tłumaczyć na Polski

Hi @Goykew 
I have sent you an invite to another repo ;-)

**Comment by BokuWaRamirezDa** [comment]
Hello! Do you need a br-portuguese translation? I am not used to gitHub, but I would like to help

**Comment by erykjj** [comment]
> Hello! Do you need a br-portuguese translation? I am not used to gitHub, but I would like to help

@BokuWaRamirezDa , thank you! I have invited you to another repo. Check your GitHub notifications ;-)

**Comment by MisaelArciniega** [comment]
i'd like to translate into spanish

**Comment by Hi-there-how-are-u** [comment]
Well why not, I can try the Swedish one here also.

**Comment by erykjj** [comment]
> Well why not, I can try the Swedish one here also.

@Hi-there-how-are-u 
Here is the link: https://crowdin.com/project/gpw/invite?h=ab0e19e296c400af3483490bcd3925b22218765

**Comment by erykjj** [comment]
@Hi-there-how-are-u 
This one is more... technical, so take your time.
Would you use the past ("completed") tense in Swedish as well?

**Comment by Hi-there-how-are-u** [comment]
> @Hi-there-how-are-u This one is more... technical, so take your time. Would you use the past ("completed") tense in Swedish as well?

I am not sure what you mean? But completed is slutförd in Swedish.

**Comment by erykjj** [comment]
I mean the tense of the expressions (in general) is "completed"/past tense:
_Suffered_ for sinners
_Cured_ sicknesses
That's all :-)

**Comment by Hi-there-how-are-u** [comment]
Oh, now I undertand. Yes past tense is used both in Swedish and Finnish.

**Comment by erykjj** [comment]
@Hi-there-how-are-u 
Here is the Swedish RC for you to check, please:
[GPW_RC_Z.zip](https://github.com/user-attachments/files/17097340/GPW_RC_Z.zip)

**Comment by erykjj** [comment]
@Hi-there-how-are-u 

Here is RC2 with the changes you made.
Also, a change of "category" ;-)

[GPW_RC2_Z.zip](https://github.com/user-attachments/files/17122002/GPW_RC2_Z.zip)

**Comment by erykjj** [comment]
So, @Hi-there-how-are-u , is GPW ready for release?

**Comment by Hi-there-how-are-u** [comment]
> So, @Hi-there-how-are-u , is GPW ready for release?

Yes, I made a few changes. But now it's ok for release!

**Comment by erykjj** [comment]
> > So, @Hi-there-how-are-u , is GPW ready for release?
> 
> Yes, I made a few changes. But now it's ok for release!

Done! Thank you again, @Hi-there-how-are-u !! Awesome job!

**Comment by Atsuson** [comment]
> Please let me know here if you can offer your help with translation on this project.

Send me the file toi translate. Thank u

**Comment by Atsuson** [comment]
> I am available to translate the text into Ewe language

**Comment by Atsuson** [comment]
> I am available to translate the text into Ewe

**Comment by Atsuson** [comment]
> Hi @Atsuson 
> Thank you, but did you finish the [BRS translation I sent you](https://github.com/erykjj/jwpublib/issues/21#issuecomment-2453052311)?
> 
> I appreciate your help, but let's finish one before starting another ;-)

I sent you the BSR file. Can I resend it to your email?

**Comment by Atsuson** [comment]
> Hi @Atsuson 
> Thank you, but did you finish the [BRS translation I sent you](https://github.com/erykjj/jwpublib/issues/21#issuecomment-2453052311)?
> 
> I appreciate your help, but let's finish one before starting another ;-)

I sent you the BSR file. Can I resend it to your email?

**Comment by erykjj** [comment]
@Atsuson 
Why do you reply twice, and in the wrong thread??
Please upload the file in the BRS translation issue: https://github.com/erykjj/jwpublib/issues/21

**Comment by Atsuson** [comment]
I have already done the translation and I don't know how to send it back to you. I have it in my sheet. Help me send it back to you

**Comment by Atsuson** [comment]
I have already done the translation and I don't know how to send it back to you. I have it in my sheet. Help me send it back to you

**Comment by erykjj** [comment]
@Atsuson 
For the last time, PLEASE, this issue is about the GPW translation. Your translation is for the BRS project. Please reply in that thread: https://github.com/erykjj/jwpublib/issues/21

![2024-11-06_10-41-51](https://github.com/user-attachments/assets/4456d549-37da-4f23-8e53-292cd92cde17)
